.github/workflows/ci-mingw.yml

```diff
@@ -0,0 +1,90 @@
+# This is a copy of ci-unix-shared-local.yml for building shared libraries
+# with an additional build configuration (using installed deps and dav1d).
+
+name: CI MINGW
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+# Cancel the workflow if a new one is triggered from the same PR, branch, or tag, except on main.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  build-mingw:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sys:
+          - mingw64
+          - mingw32
+          - ucrt64
+          - clang64
+
+    steps:
+    - uses: msys2/setup-msys2@v2
+      with:
+        msystem: ${{matrix.sys}}
+        update: true
+        install: >-
+          cmake
+          gcc
+          git
+          ninja
```

Comment:
It really is "git" only here. The rest belong to their respective environments (so in the "pacboy" option below).

Reply:
Also, instead of "gcc", do request just "cc:p" (covers both GCC and Clang via a metapackage).